test/test_stats.jl

```diff
@@ -153,8 +153,13 @@ end
 #-----------------------------------------------------------------------# Hist 
 @testset "Hist" begin
     dpdf = OnlineStats._pdf
-    #### KnownBins
-    test_exact(Hist(-5:5), y, o -> value(o)[2], y -> fit(Histogram, y, -5:5, closed=:left).weights)
+    #### FixedBins
```

Comment:
If I can be greedy, can you add one test with unequally-spaced bins?

Reply:
I've added an additional test.